cvat/schema.yml

```diff
@@ -6540,6 +6560,98 @@ components:
       description: |-
         * `random_uniform` - RANDOM_UNIFORM
         * `manual` - MANUAL
+    FunctionCall:
+      type: object
+      properties:
+        id:
+          type: string
+          description: Request id
+        function:
+          $ref: '#/components/schemas/FunctionCallParams'
+        status:
+          type: string
+        progress:
+          type: integer
+          default: 0
+        enqueued:
+          type: string
+          format: date-time
+        started:
+          type: string
+          format: date-time
+        ended:
+          type: string
+          format: date-time
+        exc_info:
+          type: string
+      required:
+      - ended
+      - enqueued
+      - exc_info
+      - function
+      - id
+      - started
+      - status
+    FunctionCallParams:
+      type: object
+      properties:
+        id:
+          type: string
+          description: The name of the function
+        task:
+          type: integer
+          description: The id of the task
+        job:
+          type: integer
+          description: The id of the job
+        threshold:
```

Comment:
This should be marked nullable, as well.

Reply:
Done